CMakeLists.txt

```diff
@@ -55,6 +55,9 @@ if(DJINNI_STATIC_LIB)
 else()
   add_library(djinni_support_lib SHARED ${SRC_SHARED})
 endif()
+
+add_library(djinni-support-lib::djinni-support-lib ALIAS djinni_support_lib)
```

Comment:
This is the name of the target that is defined if the library is installed with the conan `cmake_find_package` generator. To ensure that the target can always be linked under the same name, no matter if installed via conan or by embedding the sources in cmake directly (see updated install instructions), I have added this alias.